src/integration-test/java/mil/dds/anet/integrationtest/db/FutureEngagementWorkerTest.java

```diff
@@ -1,14 +1,15 @@
 package mil.dds.anet.integrationtest.db;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
```

Comment:
Usage of `import static org.assertj.core.api.Assertions.assertThat;` is preferred (there are still a few places left that don't, but these should be updated at some point).